package.json

```diff
@@ -293,6 +293,7 @@
     "brave-ui": "github:brave/brave-ui#5b2069a51eea6ec4577b433bb9e48b5ab2aabbe9",
     "chalk": "4.1.0",
     "commander": "2.20.3",
+    "country-flag-icons": "^1.4.20",
```

Comment:
should this be in deps, not devDeps?

Reply:
ping on the above @bsclifton - also please pin the exact version

Reply:
Working on closing w/ @nullhook 😄 We may pull this out to a separate follow-up PR and then review by itself

Reply:
Removed this code; pulled into a branch https://github.com/brave/brave-core/compare/ta-vpn-flags where we can follow up on this separately